extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/options/TlsUtils.java

```diff
@@ -66,7 +70,8 @@ public static TrustOptions computeTrustOptions(CertificateConfig certificates, O
         }
 
         if (singleTrustStoreFile != null) { // We have a single trust store file.
-            String type = certificates.trustStoreFileType.orElse(getTypeFromFileName(singleTrustStoreFile));
+            String type = certificates.trustStoreFileType.map(String::toLowerCase)
```

Comment:
changes are fine, however always calling `certificates.trustStoreFileType.map(String::toLowerCase).orElse(getTypeFromFileName("truststore", singleTrustStoreFile))` means that when file extension is not present or when truststore file type can't be matched based on the file extension, an exception is thrown, doesn't it? wouldn't `orElseGet` be better?

Reply:
ah yes, you are right!

Reply:
Fixed, in addition, I added a few tests.... because at least we have a baseline.

Reply:
(I didn't use orElseGet, as it introduces a lambda, in this case, the imperative if/then block is more efficient)

Reply:
Great, thank you.